app/controllers/organisations_controller.rb

```diff
@@ -15,6 +17,10 @@ def update
     end
   end
 
+  def show
+    @team_members = current_user&.organisation&.users || []
```

Comment:
Would the first part of this conditional be an empty array anyway?

Reply:
I think safe navigation operator returns `nil` if the object you're calling on is `nil`?

```
irb(main):001:0> something = nil
=> nil
irb(main):002:0> something&.method
=> nil
```

Reply:
Current user must have an organisation to sign in, and it will always have at least one user (the current user). So I wonder if we need any of the `&`s or the `|| []`.